plume-api/src/apps.rs

```diff
@@ -0,0 +1,13 @@
+use canapi::Endpoint;
+
+#[derive(Clone, Default, Serialize, Deserialize)]
+pub struct AppEndpoint {
+    pub id: Option<i32>,
+    pub name: Option<String>,
```

Comment:
name seems to be required by https://github.com/Plume-org/Plume/blob/36297101f261f473fd03d86a22f46379d125e002/plume-models/src/apps.rs#L47
So I think it should not be an Option